packages/flutter/test/widgets/binding_cannot_schedule_frame_test.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:typed_data';
+
+import 'package:flutter/scheduler.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/widgets.dart';
+
+void main() {
+  test('The frames will only be enabled after runApp attaches the root widget', () async {
+    final WidgetsFlutterBinding binding = WidgetsFlutterBinding.ensureInitialized() as WidgetsFlutterBinding;
```

Comment:
Why does this need to be casted to a WidgetsFlutterBinding? `ensureInitialized` returns a WidgetsBinding, which should give you access to everything you may want to call.

Reply:
I copied from the other test. will update